pattern-1/bosh-release/jobs/wso2is/templates/repository/conf/tomcat/context.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<!-- The contents of this file will be loaded for each web application -->
+<Context useHttpOnly="true" useRelativeRedirects="false" disableURLRewriting="true">
+
+    <!-- Default set of monitored resources -->
+
+    <Loader className="org.wso2.carbon.webapp.mgt.loader.CarbonWebappLoader"
+            loaderClass="org.wso2.carbon.webapp.mgt.loader.CarbonWebappClassLoader"/>
+
+    <!-- Uncomment this to disable session persistence across Tomcat restarts -->
+    <Manager
+            className="org.wso2.carbon.webapp.mgt.CarbonTomcatSessionManager"
+            sessionIdLength="128">
```

Comment:
This will create a `128 bytes` long session id. That is `256 characters` when encoded in hex (or `1024 bits`). Session ID will be a very lengthy value. Is this expected? If not the OWASP recommendation is to use minimum of `128 bit` session id, which means `16 bytes` [1]. 

It is fine to use `16 bytes` (or `32 bytes` if necessary) here but the current value is too high. Higher this value is, the more time Tomcat will take to process a fresh request (session creation time). Therefore, this will badly impact performance tests. 

[1] https://cheatsheetseries.owasp.org/cheatsheets/Session_Management_Cheat_Sheet.html#session-id-length

Reply:
128 is the value in the GA pack. Do we need to change it to 16?

Reply:
+1 to leave it as GA